InputAnswer/InputAnswer/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace InputAnswer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int excitement = 0;
+             
+
```

Comment:
Nit: Extra whitespace. There are autoformatters you can use in VS which should handle that kind of thing on save (super useful!).